commons-hpcc/src/main/java/org/hpccsystems/commons/utils/Utils.java

```diff
@@ -0,0 +1,552 @@
+package org.hpccsystems.commons.utils;
+
+/*##############################################################################
+Copyright (C) 2011 HPCC Systems.
+All rights reserved. This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as
+published by the Free Software Foundation, either version 3 of the
+License, or (at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+############################################################################## */
+
+import java.lang.reflect.Field;
+import java.text.NumberFormat;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Utils
+{
+    public static final String DOTSEPERATORREGEX = "\\.";
+
+    public static String newLine = System.getProperty("line.separator");
+    public static String fileSep = System.getProperty("file.separator");;
+
+    public final static String traceFileName = "HPCCJDBC.log";
```

Comment:
traceFileName would need to be updated

Reply:
good catch, I'll make sure no other hpccjdbc remnants made it over